deploy/metrics-apiservice.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apiregistration.k8s.io/v1beta1
+kind: APIService
+metadata:
+  name: v1alpha1.metrics
+spec:
+  insecureSkipTLSVerify: true
+  group: metrics
```

Comment:
Legacy reasons, mostly.  It's already just "metrics" in the code.  We'd probably eventually want to move to "metrics.k8s.io" when we move to beta.